src/icons/README.md

```diff
@@ -1,113 +1,56 @@
-# `fds-icons`
+### High level explanation for FDS Icons
 
-Icon library for Form Design System
+- All of our FDS Icons reflect a set of Icons that CB Insights designers have in their system. We do not accept any other outside icons.
+- These Icons live in closed source Sketch file that can only be accessed by CB Insights employees.
 
-## Getting Started
+### Making requests to add new icons to Abstract
 
-### Usage
+- In order for us to add an icon to FDS, it must be added to the UI Platform Icons set by a product designer here: https://app.abstract.com/projects/0b3bf318-e4a9-4319-8a9f-52f161094fb2/branches/master/files/D109C042-4BC3-434D-844A-6F3AA9E2881C
+- You (a CB Insights developer) can request a new icon be added to that file in this Slack channel: https://cbinsights.slack.com/archives/C01TDGQ0T47
 
-#### As a React component (recommended)
+#### Requesting icons be to FDS (for CB Insights employees)
 
-```
-import CloudIcon from '@cbinsights/fds/lib/icons/react/CloudIcon';
-
-const UploadButton = () => {
-  <button>
-    <CloudIcon />
-    Upload yourself to the cloud
-  </button>
-};
-```
-
-Every Icon component from `fds-icons` accepts the following props:
-
-| Prop        | Type                  | Default          | Description                 |
-| ----------- | --------------------- | ---------------- | --------------------------- |
-| `color`     | `String`              | `COLOR_CHARCOAL` | Color of `fill` for SVG     |
-| `size`      | `xs`,`s`,`m`,`l`,`xl` | `s`              | Standard media size of icon |
-| `className` | `String`              |                  | custom class name           |
-
-#### As raw SVG
-
-```
-import cloudSvg from '@cbinsights/fds/lib/icons/svg/Cloud.svg';
-```
-
-#### As raw PNG
+Most of you will want to request the icons be added to FDS rather than doing it yourselves:
 
-FDS provides PNGs scaled to 1x, 2x, 3x, and 4x.
-The base icon name is scaled to 1x. Specify higher res images with the `@scale` suffix:
+- Verify that the icon you need is in Abstract first: https://app.abstract.com/projects/0b3bf318-e4a9-4319-8a9f-52f161094fb2/branches/master/files/D109C042-4BC3-434D-844A-6F3AA9E2881C
+- Request that we re-run our icon export script in this channel: https://cbinsights.slack.com/archives/CL99MD596
 
-- `Cloud.png` (1x)
-- `Cloud@2x.png` (2x)
+#### Actually adding Icons to FDS
 
-```
-import cloudPng from '@cbinsights/fds/lib/icons/png/Cloud@2x.png';
-```
-
----
-
-## Development
-
-### Updating icons from a Sketch file
+To regenerate the icons, you must
 
-This project uses `sketchtool` to export SVG icons directly from a sketch file provided by the design team. You must have [Sketch](https://www.sketchapp.com/) installed in order to run the export command.
```

Comment:
Where possible I removed all insights and explanation into the process itself, and biased more step by step explanation of the process itself and what they need to do at each step.